unittests/test_parsers.py

```diff
@@ -59,3 +59,25 @@ def test_file_existence(self):
                             os.path.isfile(importer_test_file),
                             f"Unittest of importer '{importer_test_file}' is missing or using different name"
                         )
+        for parser_dir in os.scandir(os.path.join(basedir, 'dojo', 'tools')):
+            if parser_dir.is_dir():
+                for file in os.scandir(os.path.join(basedir, 'dojo', 'tools', parser_dir.name)):
+                    if file.is_file() and file.name != '__pycache__' and file.name != "__init__.py":
+                        f = os.path.join(basedir, 'dojo', 'tools', parser_dir.name, file.name)
+                        import subprocess
+                        task = subprocess.Popen(["cat", f], stdout=subprocess.PIPE)
+                        read_true = False
+                        for line in task.stdout:
+                            if read_true is True:
+                                if ('"utf-8"' in str(line) or "'utf-8'" in str(line) or '"utf-8-sig"' in str(line) or "'utf-8-sig'" in str(line)) and i <= 4:
+                                    read_true = False
+                                    i = 0
+                                elif i > 4:
+                                    self.assertTrue(False, "In file " + str(os.path.join('dojo', 'tools', parser_dir.name, file.name)) + " the test is failing because you don't have utf-8 after .read()")
```

Comment:
This might be executed inside of `with self.subTest(parser=parser_dir.name, category='read utf-8'):`. Otherwise, evaluation would stop at this moment and other files would not be checked.